app/services/partners/request_create_service.rb

```diff
@@ -18,8 +18,8 @@ def call
       @partner_request.assign_attributes(additional_attrs)
 
       unless @partner_request.valid?
-        @partner_request.errors.each do |k, v|
-          errors.add(k, v)
+        @partner_request.errors.each do |error|
+          errors.add(error.attribute, error.message)
```

Comment:
@seanmarcia similar to [above](https://github.com/rubyforgood/human-essentials/pull/2635/files#r771685406)